src/components/toc/toc.scss

```
@@ -0,0 +1,185 @@
+@use '../../style/functions.scss';
```

Stylelint (17.7.0)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)